src/js/visual/configurator-csr.js

```diff
@@ -649,7 +633,7 @@ ripe.ConfiguratorCsr.prototype._loadMesh = async function(path, format = "gltf")
         case "gltf":
             return await this._loadMeshGLTF(path);

```

Comment:
Can we move loadMeshGLTF to utils as well? We could even rename the module from "utils" to "loaders". 
PS: we can make the `useDracoLoader` an optional param to the function and pass this.useDracoLoader here

Reply:
I didn't move this one because it depends on the `useDracoLoader`, `dracoLoaderDecoderPath` and `dracoLoaderDecoderFallbackPath`.

It's implementation needs to be improved, as in drop these variables dependencies so it will be improved in the future.